### DISCUSSION TITLE
SOLVED! :) Lags of Video & Audio on Netflix - CE OS - Kodi 18.9 - NFX plugin v1.12.9 Leila

### DISCUSSION THREAD

**Discussion by neukrisz** [CastagnaIT/plugin.video.netflix#1320] [General]
Dear Community,
I have a problem with playing movies from Neflix :(
Many interruptions in video stream (and audio as well)

HW:
Minix Neo U1 (huge lags)
Odroid C2 (huge lags)
VIM3L (minor lags)

OS:
CoreElec v9.2.8 
KODI: 18.9 Leila
Installed components:
repository.castagnait-1.0.1.zip
CastagnaIT/plugin.video.netflix v1.12.9-Leia

I tested all three hardware in different internet connection / different places - reinstall (fresh) CE OS too.
The problem is existing everywhere - for me and my two friend too.

Please help me fix this.
Many thanks!

**Comment by neukrisz** [comment]
### **THE SOLUTION is the following:**
This method worked fine for us:
"Install any add-on from Slyguy Repo (https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html)
And then go to Add-ons > services > slyguy common > advanced > re(install) widevine CDM
Then select 4.10.2252.5"

**The picture is fluent again ;)**

Thanks for the suggestion for [CastagnaIT](https://github.com/CastagnaIT)

**Comment by Daikath** [comment]
Thanks, I was trying to get netflix to play, so I could fall asleep to it last night.. I couldn't get it to work, I kept thinking that my video was in pal, but the audio was in ncts..., since it was in sync at the beginning of a video, but got more out of sync as it went on.. Thankfully this worked perfectly.

**Comment by ostanislaw** [comment]
Hi,

I think I had exactly same issue like you, and I'm using latest widevine. Earlier watched at rpi3, and it was lagging at 720p. But then tried Odroid N2+ (that's best HW in terms of ARM CPU). And it was not much better.
But yesterday I was playing with different settings and after:
**disabling VP9 profiles** the video is completely smooth and in sync with audio. I was even able to play at 1080p also completely smootly. Give it also a try. Maybe selecting older widevine has the same effect as it might not support someting.

Could someone verify it? Maybe it should be disabled by default?
